tools/repology.json

```diff
@@ -13135,6 +13111,14 @@
     "license": "BSD-2",
     "compatibility": "all"
   },
+  {
+    "name": "py3_pygobject",
```

Comment:
Do these py3_ names need to be changed?

Reply:
this is the packagae you renamed

Reply:
Yes, but does Repology want the actual pypi name vs something prefixed with `py3_`?

Reply:
they handle this themselves https://repology.org/project/python:pygobject/versions